tapdb/assets_store_test.go

```diff
@@ -2204,6 +2205,44 @@ func TestTransferOutputProofDeliveryStatus(t *testing.T) {
 	require.Equal(
 		t, randBlockHash[:], assetTransfers[0].AnchorTxBlockHash,
 	)
+
+	// Let's insert a burn.
+	assetID := inputAsset.ID()
+	transferID := assetTransfers[0].ID
+	_, err = assetsStore.db.InsertBurn(ctx, sqlc.InsertBurnParams{
```

Comment:
We'll want to test the actual `AssetStore`'s `QueryBurns` and `InsertBurn` methods, not directly call into the DB-level methods.